### ISSUE TITLE
As a user I would like an 'About Us' page to learn more about Providence Geeks

### ISSUE THREAD

**Issue by saimatsumoto** [ProvidenceGeeks/website-backlog#7]
## Type of Change
- [x] New Feature Request
- [x] Improvement / Suggestion
- [x] Other (please clarify below)

## Summary
* From our discussion from Demo/Kick-off on 10/23, there was a suggestion on adding 'About Us'
- [ ] Evaluate the need for this feature
- [ ] Once evaluation is done, create a new issue in respective repositories.
